check-kernel-config

```diff
@@ -211,6 +211,16 @@ CONFIG_EPOLL
 CONFIG_SIGNALFD
 CONFIG_TIMERFD
 CONFIG_TMPFS_POSIX_ACL
+CONFIG_VETH
+CONFIG_SW_SYNC_USER
```

Comment:
This seems to have been removed from upstream kernels after 4.5. Is it still being carried in Android trees?

Reply:
Idk, it was presented to me as a Waydroid requirement, @NotKit may know it, or Erfan himself.